src/ui/Pager/Pager.ts

```diff
@@ -248,18 +251,26 @@ export class Pager extends Component {
       this.currentPage = pagerBoundary.currentPage;
       if (pagerBoundary.end - pagerBoundary.start > 0) {
         for (let i = pagerBoundary.start; i <= pagerBoundary.end; i++) {
-          const listItemValue = document.createElement('a');
-          $$(listItemValue).addClass(['coveo-pager-list-item-text', 'coveo-pager-anchor']);
-          $$(listItemValue).text(i.toString(10));
+          const listItemValue = $$(
+            'a',
+            {
+              className: 'coveo-pager-list-item-text coveo-pager-anchor',
+              tabindex: -1,
+              ariaHidden: 'true'
+            },
+            i.toString(10)
+          ).el;
 
           const page = i;
           const listItem = $$('li', {
             className: 'coveo-pager-list-item',
             tabindex: 0
           }).el;
-          if (page === this.currentPage) {
+          const isCurrentPage = page === this.currentPage;
+          if (isCurrentPage) {
             $$(listItem).addClass('coveo-active');
```

Comment:
Do we have a mechanism to remove the `coveo-active` class when we click on a different page? Or are we always re-building the elements?

Same comment for the Results Per Page component,

Reply:
I haven't found any mechanism that removes the `coveo-active` class from either component.